src/dune_util/global_lock.mli

```diff
@@ -3,9 +3,11 @@
     Before starting rpc, writing to the build dir, this lock should be locked. *)
 
 module Lock_held_by : sig
-  type t =
-    | Pid_from_lockfile of int
-    | Unknown
+  type t
+
+  (** returns "(pid: X)" where X is the PID if the PID is known, otherwise the
```

Comment:
It actually returns a space before the paren. But I think it would sort of be nicer to just return `pid: X` or something and then let the formatter fit it into the formatting string.

Reply:
I've updated the comment to include the leading space. I agree with the sentiment about just returning `pid: X` but looking at how this function is actually used, it's always to produce a message like `another dune instance (pid: X) has the lock` if the pid is known and a message `another dune instance has the lock` otherwise. I'm trying to reduce the repetition when producing messages like this. I don't think my solution is super elegant which is why I initially just exposed the variant type but I found that I was repeating the logic for constructing the string a lot.